types/googlemaps/reference/marker.d.ts

```diff
@@ -458,6 +458,17 @@ declare namespace google.maps {
      * @see {@link https://developers.google.com/maps/documentation/javascript/reference/marker#MarkerLabel Maps JavaScript API}
      */
     interface MarkerLabel {
+        /**
+         * The className property of the label's element (equivalent to the element's class attribute).
+         * Multiple space-separated CSS classes can be added. Default is no CSS class (an empty string).
+         * The font color, size, weight, and family can only be set via the other properties of MarkerLabel.
+         * CSS classes should not be used to change the position nor
+         * orientation of the label (e.g. using translations and rotations) if also using marker collision management.
+         * @default ''
+         * @see {@link https://developers.google.com/maps/documentation/javascript/reference/marker#MarkerLabel.className Maps JavaScript API}
+         * @see {@link https://developers.google.com/maps/documentation/javascript/examples/marker-collision-management marker collision management}
```

Comment:
How do you look at leaving the link where it was in the documentation?

```suggestion
         * CSS classes should not be used to change the position nor orientation of the label (e.g. using translations
         * and rotations) if also using marker {@link https://developers.google.com/maps/documentation/javascript/examples/marker-collision-management marker collision management}.
         * @default ''
         * @see {@link https://developers.google.com/maps/documentation/javascript/reference/marker#MarkerLabel.className Maps JavaScript API}
```

Reply:
Sounds good!